internal/access/identity_test.go

```diff
@@ -81,6 +82,13 @@ func TestDeleteIdentityCleansUpResources(t *testing.T) {
 	err = data.CreateGrant(db, grantDestination)
 	assert.NilError(t, err)
 
+	err = data.CreateGroup(db, &models.Group{Name: "Group"})
+	assert.NilError(t, err)
+	groups, err := data.ListGroups(db, nil)
+	assert.NilError(t, err)
+	err = data.AddUsersToGroup(db, groups[0].ID, []uid.ID{identity.ID})
```

Comment:
I think you can get the `group.ID` this way without needing to list:

```suggestion
	group := &models.Group{Name: "Group"}
	err = data.CreateGroup(db, group)
	assert.NilError(t, err)
	err = data.AddUsersToGroup(db, group.ID, []uid.ID{identity.ID})
```

This works because `CreateGroup` sets the ID on the passed in pointer, so we can access it after the call to `CreateGroup`.